### PR TITLE
doc: Fix install.md code blocks

### PR DESCRIPTION
**Pull request by mbatra-hunter**
I noticed that the code blocks in INSTALL.md weren't formatted right, so I fixed it

### CODE REVIEW

**Review by AndreasFuchsTPM — APPROVED** [review]

**Review by AndreasFuchsTPM — CHANGES_REQUESTED** [review]
Please add a "signed off by" tag to the commit message.

**Comment by mbatra-hunter** [comment]
Thank you! I didn't know if I could amend an existing commit in a PR but your snippet helped. I was going to look into finding out how but didn't get around to it.
I think everything's fine now, all the workflows pass

**Review by AndreasFuchsTPM — APPROVED** [review]

File: INSTALL.md [hunk 1]
```diff
@@ -263,4 +263,4 @@ make
 * opensuse-leap
 * ubuntu-22.04
 * alpine-3.15
-* FreeBSD
\ No newline at end of file
+* FreeBSD
```
Comment (AndreasFuchsTPM) [inline INSTALL.md:266]: Maybe I'm stupid, but what is the change in this line ?!?

Comment (mbatra-hunter) [reply INSTALL.md:266]: A newline was either added or removed at the end of the file. It was on accident by me, I was using the GitHub web editor and it didn't show it as a change until after I opened my PR

**Pull request merged**